src/discussion/serializers.py

```diff
@@ -164,9 +166,27 @@ def get_post(self, thread):
         )
         return serializer.data
 
+    def get_review(self, obj):
+        if obj.review:
+            context = self.context
+            _context_fields = context.get("dis_dts_get_review", {})
+
+            serializer = DynamicReviewSerializer(
+                obj.review,
+                context=context,
+                **_context_fields
+            )
+
+            return serializer.data
+
+        return None
+
     def get_peer_review(self, obj):
```

Comment:
Note that we  will need to purge/refactor codebase of peer_review once we figure out next steps